okapi-core/src/main/resources/log4j2-dev.properties

```diff
@@ -0,0 +1,18 @@
+status = error
```

Comment:
Adam said that production default should be JSONLayout and development default should be PatternLayout for readability.
Can you copy okapi-core/src/main/resources/log4j2-dev.properties to  okapi-core/src/test/resources/log4j2-test.properties?